components/studio/bin/hab-studio.ps1

```diff
@@ -231,6 +231,7 @@ function New-Studio {
   )
 
   $env:PATH = [String]::Join(";", $pathArray)
+  $env:PSModulePath = "$PSScriptRoot\powershell\Modules"
```

Comment:
Similarly, is it significant that this is "PSModulePath", while above we have "PSMODULEPATH"?

Reply:
same as above applies. Generally `PSModulePath` is used and what is automatically set but I kept it uppercase above just to make it look like the others. To windows/powershell they are all the same.

Reply:
Same suggestion above re: consistency... if `PSModulePath` is customary in Powershell world, we should probably stick with that.